content/200-concepts/100-components/01-prisma-schema/04-data-model.mdx

```diff
@@ -419,13 +634,23 @@ See [complete list of field and block attributes](../../../reference/api-referen
 
 ### Defining an ID field
 
-An ID uniquely identifies individual records of a model. IDs can be defined by a single field using the [`@id`](../../../reference/api-reference/prisma-schema-reference#id) <span class="api"></span> attribute, or multiple fields using the [`@@id`](../../../reference/api-reference/prisma-schema-reference#id-1) <span class="api"></span> attribute.
+An ID uniquely identifies individual records of a model. A model can only have _one_ ID:
 
-In the following example, the `User` ID is represented by the `id` field:
+  * In **relational databases**, the ID can be a single field with or based on multiple fields. If a model does not have an define an `@id` or an `@@id`, you must define a manatory `@unique` field or `@@unique` block instead.
```

Comment:
```suggestion
  * In **relational databases**, the ID can be a single field with or based on multiple fields. If a model does not hav an `@id` or an `@@id`, you must define a mandatory `@unique` field or `@@unique` block instead.
```
Is the second sentence necessary here though? I know this is needed for Prisma to work with the model, but that is not about "Defining an ID field" which is the headline here.

Reply:
I think it's necessary - if you don't define an ID, you have to define a @unique that acts like your ID.

No longer applies to Mongo; was a feature before but not any more - removed.